morphocut/str.py

```diff
@@ -86,6 +88,8 @@ class Parse(Node):
             with Pipeline() as pipeline:
                 result = Parse(fmt, string, case_sensitive)
 
+    This returns the following key-value output: {'named': 'TEST'}
+
```

Comment:
Indent by 4 spaces.

`` Result: `obj[result]` == `{'named': 'TEST'}` for a stream object `obj`.``